ui/src/components/RemoteAccess/PassControlDialog.jsx

```diff
@@ -40,10 +41,12 @@ function PassControlDialog() {
   }
 
   useEffect(() => {
-    if (!showModal) {
+    if (showModal) {
+      dispatch(hideWaitDialog()); // avoid conflict with any yet-to-be-dismissed dialog
```

Comment:
Obviously less than ideal. It would be better if the Redux state held a single dialog visibility state but this isn't trivial.